devtools/run-tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -ev
+
+BACKEND_CHANGE=`git diff ../src/backend`
```

Comment:
I don't understand how this can works, it open diif tool with the diff between HEAD and HEAD.

what do you think of the command I provided: 

```
git diff-tree --no-commit-id --name-only -r HEAD..master src/frontend
```

it shows a list of changed files.  
`master` and `src/frontend` can be defined with var obviously